.gitmodules

```diff
@@ -6,3 +6,6 @@
 [submodule "user-rating"]
 	path = user-rating
 	url = https://github.com/backstop-protocol/user-rating.git
+[submodule "openzeppelin-upgrades"]
```

Comment:
can we remove this now?

Reply:
We are using `Initializable` to allow initialization only once.